addon/webextension/background/main.js

```diff
@@ -14,6 +14,9 @@ window.main = (function () {
 
   browser.storage.local.get(["hasSeenOnboarding"]).then((result) => {
     hasSeenOnboarding = !! result.hasSeenOnboarding;
+    if (! hasSeenOnboarding) {
+      setIconActive(false, null);
```

Comment:
Technically we could leave it off and it'll default to undefined right now!  But I don't know, how can you know what's right in JavaScript, amiright?